sys/net/network_layer/ipv6/addr/ipv6_addr.c

```diff
@@ -106,6 +111,32 @@ void ipv6_addr_init_iid(ipv6_addr_t *out, const uint8_t *iid, uint8_t bits)
     memcpy(&(out->u8[pos]), iid, bytes);
 }
 
+int ipv6_addr_split(char *addr_str, char seperator, int _default)
+{
+    char *sep = addr_str;
+    while(*++sep) {
+        if (*sep == seperator) {
+            *sep++ = '\0';
+            if (*sep) {
+                _default = atoi(sep);
+            }
+            break;
+        }
+    }
+
+    return _default;
+}
+
+#ifdef MODULE_FMT
+void print_ipv6_addr(const ipv6_addr_t *addr)
```

Comment:
btw, I like the idea of having this function. But you did not use it in this Pr or in #4754. Was that intentional?